dist/Storable/Storable.xs

```diff
@@ -5882,6 +5898,50 @@ static SV *retrieve_integer(pTHX_ stcxt_t *cxt, const char *cname)
     return sv;
 }
 
+/*
+ * retrieve_boolean_true
+ *
+ * Retrieve boolean true copy.
+ */
+static SV *retrieve_boolean_true(pTHX_ stcxt_t *cxt, const char *cname)
+{
+    SV *sv;
+    HV *stash;
+
+    TRACEME(("retrieve_boolean_true (#%d)", (int)cxt->tagnum));
+
+    sv = newSVsv(&PL_sv_yes);
```

Comment:
I *was* going to comment that there is a new `newSVbool` function you can use here, but then writing it this way means this part of the code is portable to older perls before core bools anyway and still does (as close to) the right thing there. So this is fine.